Add missing const in deleted copy constructor (from 1ed01f1e1)

This fails only with g++<9, probably because of P0641R2 which was added in
g++-9.

diff --git codels/camera.h codels/camera.h
index fb30f51..20a2e26 100644
--- codels/camera.h
+++ codels/camera.h
@@ -42,7 +42,7 @@ struct camera_job {
   enum state { idle, update, cancel };
 
   camera_job() {};
-  camera_job(camera_job &) = delete;
+  camera_job(const camera_job &) = delete;
   void operator=(const camera_job &) = delete;
   virtual ~camera_job();
   virtual void destroy() = 0;
